core/src/main/scala/kafka/server/BrokerLifecycleManager.scala

```diff
@@ -53,7 +53,8 @@ import scala.jdk.CollectionConverters._
  */
 class BrokerLifecycleManager(val config: KafkaConfig,
```

Comment:
can you use the standard style of
```
foo(
  bar
  baz
  quux
) {
}
```

this code was written before we adopted that but if we're adding new parameters let's format it that way to avoid having so much indentation